### ISSUE TITLE
Add support for Rollups API

### ISSUE THREAD

**Issue by gmuehlenberg** [kairosdb/kairosdb-client#66]
As of now, the Kairosdb-Client is a bit incomplete because it provides no support for the management of Rollups. The company I am working for needs rollups to create preaggregated data, so I will need to create some classes for this. Do you also see this as a sensible extension for the kairosdb-client?

**Comment by jsabin** [comment]
Are you suggesting a client API that hooks into the Kairos [roll-up REST APIs](http://kairosdb.github.io/docs/build/html/restapi/Roll-ups.html) or are you talking about aggregating the data before you push into Kairos? The term "rollup" is somewhat overloaded.

**Comment by gmuehlenberg** [comment]
I am suggesting to use the roll-up REST API.

**Comment by jsabin** [comment]
Yep makes sense. I will add this to the client. Thanks for the suggestion.

**Comment by gmuehlenberg** [comment]
If you wish, I might as well contribute this.
